src/runtime/runtime.cs

```diff
@@ -297,16 +297,20 @@ internal static void Initialize(bool initSigs = false)
             IntPtr dllLocal = IntPtr.Zero;
             var loader = LibraryLoader.Get(OperatingSystem);
```

Comment:
Seems like you should delete these lines too, and line 295 as well since you moved it below.